guides/common/modules/proc_configuring-server-for-use-with-tftp.adoc

```diff
@@ -1,41 +1,53 @@
-[id="configuring-external-tftp_{context}"]
-= Configuring {ProductName} with external TFTP
+[id="configuring-server-for-use-with-tftp"]
+= Configuring {ProductName} for use with tftp
 
 You can configure {ProductName} with external TFTP services.
 
+
+.Prerequisites
+* You configured the TFTP server.
+* You shared the `/var/lib/tftpboot` on the TFTP server with NFS.
+
+
 .Procedure
+. Associate the TFTP service with the appropriate subnets.
+
+. For each affected {SmartProxy}, update the configuration of that {SmartProxy} in the {ProjectWebUI}. See xref:associating-the-tftp-service-with-a-subnet[].
+
 . Create the TFTP directory for NFS:
 +
 [options="nowrap"]
 ----
 # mkdir -p /mnt/nfs/var/lib/tftpboot
 ----
+
 . In the `/etc/fstab` file, add the following line:
 +
 [options="nowrap" subs="+quotes"]
 ----
 _TFTP_Server_IP_Address_:/exports/var/lib/tftpboot /mnt/nfs/var/lib/tftpboot nfs rw,vers=3,auto,nosharecache,context="system_u:object_r:tftpdir_rw_t:s0" 0 0
```

Comment:
Future improvement: we should review this mount line. I don't think there's a reason to use version 3.

Reply:
I had the same idea to switch to v4 when I read the hard-coded v3 in one of the other sections yesterday.